pages/accountLists/[accountListId]/settings/organizations.graphql

```diff
@@ -10,3 +10,11 @@ query Organizations {
     }
   }
 }
+
+mutation UpdateUser($input: UserUpdateMutationInput!) {
+  updateUser(input: $input) {
+    user {
+      setup
```

Comment:
You're going to want to add the `user`'s `id` field so that the cache gets updated properly.

Reply:
FYI, this field is different from what you're setting with `UpdateUserOptions`. Instead of refreshing the `setup` field, you might want to refresh the `userOptions` field.